app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -198,6 +201,7 @@ public Message(ZulipApp app, JSONObject message,
 
         this.setTimestamp(new Date(message.getLong("timestamp") * 1000));
         this.setMessageRead(false);
+        this.setMessageStar(false);
```

Comment:
@kunall17 I was also confused about how to get value of star in response(while implementing #290 (Part 2)). I started reading Zulip docs but couldn't find how to get star value in response. So, I left it for some time and started  solving issue #290. 

Please give me a hint about how to do it?